README.md

```diff
@@ -1,7 +1,16 @@
 ## Collaboration method for processing the survey data
 
 1. Clone this repo
-2. Download the latest spreadsheet from [sharepoint](https://dmail.sharepoint.com/:f:/r/sites/SATRE/Shared%20Documents/WP1%20-%20Design/[wip](https://dmail.sharepoint.com/:f:/r/sites/SATRE/Shared%20Documents/WP1%20-%20Design/wip?csf=1&web=1&e=baCrJ9)?csf=1&web=1&e=baCrJ9) and put it in the the local copy of the repo
+2. Download the latest spreadsheet from [sharepoint](<https://dmail.sharepoint.com/:f:/r/sites/SATRE/Shared%20Documents/WP1%20-%20Design/[wip](https://dmail.sharepoint.com/:f:/r/sites/SATRE/Shared%20Documents/WP1%20-%20Design/wip?csf=1&web=1&e=baCrJ9)?csf=1&web=1&e=baCrJ9>) and put it in the the local copy of the repo
 3. Build the conda environment
-4.  Use `load_survey_data.load_data` to get a dataframe that can be easily used for further manipulations (see `summary-so-far.ipynb`)
-5. Run `pre-commit install` in the local copy of the repo to ensure the git hook for stripping notebook output is installed. Note that it operates on `git commit`, and removes all output from the local copy of the notebook
\ No newline at end of file
+4. Use `load_survey_data.load_data` to get a dataframe that can be easily used for further manipulations (see `summary-so-far.ipynb`)
+5. Run `pre-commit install` in the local copy of the repo to ensure the git hook for stripping notebook output is installed. Note that it operates on `git commit`, and removes all output from the local copy of the notebook
+
+## Data files
+
+The original survey results contain sensitive data, but a sanitised version is available:
+
+- [`data/public-results-for-satre-tre-op-2023-05-16-1359.csv`](data/public-results-for-satre-tre-op-2023-05-16-1359.csv) contains the ranked answers from the survey, along with:
```

Comment:
To my surprise, the GitHub review interface opens these links on the on the same branch as this PR :rocket:!